.changeset/lovely-llamas-retire.md

```diff
@@ -0,0 +1,84 @@
+'@commercetools-uikit/field-errors': minor
+'@commercetools-uikit/field-label': minor
+'@commercetools-uikit/async-creatable-select-field': minor
+'@commercetools-uikit/async-select-field': minor
+'@commercetools-uikit/creatable-select-field': minor
+'@commercetools-uikit/date-field': minor
+'@commercetools-uikit/date-range-field': minor
+'@commercetools-uikit/date-time-field': minor
+'@commercetools-uikit/localized-multiline-text-field': minor
+'@commercetools-uikit/localized-text-field': minor
+'@commercetools-uikit/money-field': minor
+'@commercetools-uikit/multiline-text-field': minor
+'@commercetools-uikit/number-field': minor
+'@commercetools-uikit/password-field': minor
+'@commercetools-uikit/radio-field': minor
+'@commercetools-uikit/search-select-field': minor
+'@commercetools-uikit/select-field': minor
+'@commercetools-uikit/text-field': minor
+'@commercetools-uikit/time-field': minor
+'@commercetools-uikit/grid': minor
+'@commercetools-uikit/icons': minor
+'@commercetools-uikit/async-creatable-select-input': minor
+'@commercetools-uikit/async-select-input': minor
+'@commercetools-uikit/checkbox-input': minor
+'@commercetools-uikit/creatable-select-input': minor
+'@commercetools-uikit/date-input': minor
+'@commercetools-uikit/date-range-input': minor
+'@commercetools-uikit/date-time-input': minor
+'@commercetools-uikit/input-utils': minor
+'@commercetools-uikit/localized-money-input': minor
+'@commercetools-uikit/localized-multiline-text-input': minor
+'@commercetools-uikit/localized-rich-text-input': minor
+'@commercetools-uikit/localized-text-input': minor
+'@commercetools-uikit/money-input': minor
+'@commercetools-uikit/multiline-text-input': minor
+'@commercetools-uikit/number-input': minor
+'@commercetools-uikit/password-input': minor
+'@commercetools-uikit/radio-input': minor
+'@commercetools-uikit/rich-text-input': minor
+'@commercetools-uikit/rich-text-utils': minor
+'@commercetools-uikit/search-select-input': minor
+'@commercetools-uikit/select-input': minor
+'@commercetools-uikit/select-utils': minor
+'@commercetools-uikit/text-input': minor
+'@commercetools-uikit/time-input': minor
+'@commercetools-uikit/toggle-input': minor
+'@commercetools-uikit/label': minor
+'@commercetools-uikit/link': minor
+'@commercetools-uikit/loading-spinner': minor
+'@commercetools-uikit/messages': minor
+'@commercetools-uikit/notifications': minor
+'@commercetools-uikit/primary-action-dropdown': minor
+'@commercetools-uikit/spacings-inline': minor
+'@commercetools-uikit/spacings-inset': minor
+'@commercetools-uikit/spacings-inset-squish': minor
+'@commercetools-uikit/spacings-stack': minor
+'@commercetools-uikit/stamp': minor
+'@commercetools-uikit/table': minor
+'@commercetools-uikit/tag': minor
+'@commercetools-uikit/text': minor
+'@commercetools-uikit/tooltip': minor
+'@commercetools-uikit/utils': minor
+'visual-testing-app': minor
+---
+
+Migrate to emotion v11. https://emotion.sh/docs/emotion-11
```

Comment:
I don't think we need to bump a major version here. The emotion packages are also not peer dependencies, so there should be no need for a major.